conf/application.conf

```diff
@@ -57,6 +57,9 @@ aws {
   s3.bucket.upload.quarantine = file-upload-quarantine
   s3.bucket.upload.quarantine = ${?S3_BUCKET_QUARANTINE}
   s3.bucket.env-subdir = local
+  s3.timeout.connection = 5 seconds
+  s3.timeout.request = 19 seconds
```

Comment:
Durring biggest outage I've seen some timeouts like 20s, so I prefer to finish before them.